common/src/main/java/com/graphicsfuzz/common/util/ShaderJobFileOperations.java

```diff
@@ -156,18 +156,26 @@ public boolean areImagesOfShaderResultsInteresting(
 
   public boolean areShadersValid(
       File shaderJobFile,
-      boolean throwExceptionOnInvalid)
+      boolean throwExceptionOnInvalid,
+      boolean vulkan)
```

Comment:
vulkan -> isVulkan

Much as I don't really like JavaDoc, I do think this one could do with a JavaDoc comment in order to explain that the `isVulkan` parameter is used to instruct the validator to work in Vulkan-aware mode.

Reply:
done